redbot/cogs/mutes/mutes.py

```diff
@@ -831,7 +833,7 @@ async def force_role_mutes(self, ctx: commands.Context, true_or_false: bool):
         if true_or_false:
             await ctx.send(_("Okay I will enforce role mutes before muting users."))
         else:
-            await ctx.send(_("Okay I will allow channel overwrites for muting users."))
+            await ctx.send(_("Okay I will use timeouts for muting users."))
```

Comment:
I don't think this makes it use timeouts, it just allows timeouts to be used at all. I'm not sure if this setting is even needed, now that there's no API spam to need to opt in to.